src/Uno.UI.Runtime.Skia.MacOS/README.md

```diff
@@ -1,5 +1,14 @@
 # Skia Host for macOS
 
+## Preparation
+
+Before you can build you need the libSkiaSharp.dylib.
+```bash
+cd UnoNativeMac
+./getSkiaSharpDylib.sh
+```
+See /UnoNativeMac/README.md for more info.
```

Comment:
This could be a link to the other README.md

Reply:
You're right - done